.yamllint

```diff
@@ -0,0 +1,9 @@
+---
+
+extends: default
+
+rules:
+  line-length:
+    level: warning
+    max: 120
```

Comment:
from 79 to 120 now??

Reply:
Mostly from an error to a warning because for yaml files the formatting can really become a lot worse if you shorten the lines. We already had this limit in ESMValTool (for those long dataset entries in recipes), so that's why I copied it here. Happy to change it to 79 if you think that's more consistent though.

Reply:
mrh up to you. I thought the linter error changed, and now we'll see 120 char lines everywhere :grin:

Reply:
https://github.com/ESMValGroup/ESMValTool/blob/main/.yamllint

Reply:
OK, proves out I am talking nonsense :laughing: